src/ftsh_functions.c

```diff
@@ -74,9 +114,9 @@ int ftsh_execute(char **args)
 
     for (i = 0; i < ftsh_num_builtins(); i++) {
         if (strcmp(args[0], builtin_str[i]) == 0) {
-            return (*builtin_func[i])(args);
+            return (*builtin_func[i])(args, pf);
         }
     }
 
-    return ftsh_launch(args);
-}
\ No newline at end of file
+    return 0;
```

Comment:
Let's have this function return `EXIT_SUCCESS` and `EXIT_FAILURE`.

Reply:
changed